src/CardSection/CardSection.js

```diff
@@ -5,31 +5,47 @@ import { Link } from 'react-browser-router'
 import DetailsPage from '../DetailsPage/DetailsPage'
 
 function CardSection(props) {
-  let movies;
-  if (props.allMovies.length > 0) {
-    movies = props.allMovies.map(movie => {
+  console.log('Your favs', props)
+  // const filtered = props.filterFavs()
+
+  let movies, favoriteMovie;
```

Comment:
I love that is is dynamic to accept all movies or favorites based on user interaction.